theano/tensor/nnet/tests/test_abstract_conv.py

```diff
@@ -0,0 +1,18 @@
+import unittest
+from theano.tensor.nnet.abstract_conv import get_conv_output_shape
+
+
+class TestGetConvOutShape(unittest.TestCase):
+
+    def test_basic(self):
+        image_shape, kernel_shape = (3, 2, 8, 8), (4, 2, 5, 5)
```

Comment:
This is a good start. To make the test coverage better, test with non square image and filter (change the last digit in the 2 shapes above).